resources/views/searches/resources/groups/add_group.blade.php

```diff
@@ -0,0 +1,96 @@
+<!-- Add group button - OPEN -->
+
```

Comment:
The button missing

Reply:
I removed the comment. The addGroupModal now only contains the Modal screen.  It is called by 2 buttons (lines 12 and 40 of groups/index.blade.php.